index.html

```diff
@@ -23,6 +29,14 @@ <h1 class="heading">Екатерина Фомичева</h1>
 
     <section class="intro">
       <div class="container">
+        <figure class="user-image">
```

Comment:
user image should be placed **under** subheading

Reply:
fixed